packages/reactstrap-validation-select/AvResourceSelect.js

```diff
@@ -164,6 +168,11 @@ class AvResourceSelect extends Component {
       .catch(() => ({ options: [], hasMore: false }));
   };
 
+  onFocusHandler = (...args) => {
+    this.setState({ dropdownHasBeenFocused: true });
+    if (this.props.onFocus) this.props.onFocus(...args);
```

Comment:
Can we add this to `ResourceSelect` as well?

Reply:
should already be there. Line 42

Reply:
🙈